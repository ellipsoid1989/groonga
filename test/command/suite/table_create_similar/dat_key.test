table_create Data TABLE_DAT_KEY Int32

table_create_similar DataSimilar Data

dump
