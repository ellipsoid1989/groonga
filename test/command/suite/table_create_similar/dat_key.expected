table_create Data TABLE_DAT_KEY Int32
[[0,0.0,0.0],true]
table_create_similar DataSimilar Data
[[0,0.0,0.0],true]
dump
table_create Data TABLE_DAT_KEY Int32

table_create DataSimilar TABLE_DAT_KEY Int32
